x/auth/client/testutil/suite.go

```diff
@@ -428,6 +428,76 @@ func (s *IntegrationTestSuite) TestCLIQueryTxCmdByEvents() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestCLIQueryTxsCmdByEvents() {
```

Comment:
Took this from the `QueryTxCmdByEvents` and swapped out the command being executed, test cases, and the logic when checking the happy case